src/main/java/com/minecolonies/coremod/items/ModItems.java

```diff
@@ -41,12 +43,24 @@ public ModItems()
      */
     public static void init(final IForgeRegistry<Item> registry)
     {
-        registry.register(supplyChest = new ItemSupplyChestDeployer());
-        registry.register(buildTool = new ItemBuildTool());
-        registry.register(scanTool = new ItemScanTool());
-        registry.register(permTool = new ItemScepterPermission());
-        registry.register(caliper = new ItemCaliper());
-        registry.register(scepterGuard = new ItemScepterGuard());
-        registry.register(supplyCamp = new ItemSupplyCampDeployer());
+        supplyChest = new ItemSupplyChestDeployer();
+        buildTool = new ItemBuildTool();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Synchronize this lazy initialization of 'buildTool' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2444)